test/jdk/javax/swing/JEditorPane/bug4325606.java

```diff
@@ -97,7 +97,7 @@ public void mouseClicked(MouseEvent event) {
             try {
                 Utilities.getRowStart(pane, pane.getCaretPosition());
             } catch (BadLocationException blex) {
-                passed = false;
+                throw new RuntimeException("Test failed.");
```

Comment:
```suggestion
                throw new RuntimeException("Test failed.", blex);
```
Preserve the original exception as it shows where the problem occurred.

Reply:
The only thing left… I consider it important.